agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -2001,13 +2001,17 @@ bool backhaul_manager::handle_1905_discovery_query(ieee1905_1::CmduMessageRx &cm
                         auto client_mac       = std::get<0>(associated_client);
                         auto association_time = std::get<1>(associated_client);
 
-                        auto elapsed = std::chrono::duration_cast<std::chrono::seconds>(
-                            now - association_time);
+                        auto elapsed =
+                            std::chrono::duration_cast<std::chrono::seconds>(now - association_time)
+                                .count();
+                        if (elapsed > UINT16_MAX) {
+                            elapsed = UINT16_MAX;
+                        }
```

Comment:
For completeness, we should also handle the `elapsed < 0` case.

Reply:
Done